fraud_batch/3_model_training.ipynb

```diff
@@ -211,55 +211,6 @@
     "predictions"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "acb580b5",
-   "metadata": {
-    "jp-MarkdownHeadingCollapsed": true,
-    "tags": []
-   },
-   "source": [
-    "## <span style=\"color:#ff5f27;\"> 👓  Exploration</span>\n",
-    "In the Hopsworks feature store, the metadata allows for multiple levels of explorations and review. Here you will explore a few of those capacities. \n",
-    "\n",
-    "### 🔎 <b>Search</b> \n",
-    "Using the search function in the UI, you can query any aspect of the feature groups, feature_view and training data that was previously created.\n",
-    "\n",
-    "### 📊 <b>Statistics</b> \n",
-    "You can also enable statistics in one or all the feature groups."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "e21da9c3",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "trans_fg = fs.get_feature_group(\"transactions_fraud_batch_fg\", version = 1)\n",
-    "trans_fg.statistics_config = {\n",
-    "    \"enabled\": True,\n",
-    "    \"histograms\": True,\n",
-    "    \"correlations\": True\n",
-    "}\n",
-    "\n",
-    "trans_fg.update_statistics_config()\n",
-    "trans_fg.compute_statistics()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "dd07a75b",
-   "metadata": {},
-   "source": [
-    "![fg-statistics](../images/fg_statistics.gif)\n",
-    "\n",
-    "\n",
-    "### ⛓️ <b> Lineage </b> \n",
-    "In all the feature groups and feature view you can look at the relation between each abstractions; what feature group created which training dataset and that is used in which model.\n",
-    "This allows for a clear undestanding of the pipeline in relation to each element. "
-   ]
-  },
```

Comment:
Statistics for exploratory data analysis is a key component of the platform. The tutorials should show that. I think it was originally put here because we didn't  want to block  users to make progress. If you want to move it back to the feature group notebook that's fine, but we need to have it somewhere.